src/opengeodeweb_viewer/vtkw_server.py

```diff
@@ -94,10 +96,12 @@ def run_server():
 
     _Server.add_arguments(parser)
     args = parser.parse_args()
-    args.port = os.environ.get("PORT")
+    if not "port" in args or args.port == 8080:
```

Comment:
Why do you discard port 8080?

Reply:
Port is already in the default argments and if I don't specify that is run on 8080, and I don't want any breaking change